packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig

```diff
@@ -22,12 +25,12 @@
         <ol>
             {% for item in _items %}
                 {% if loop.index is same as(_items|length - 1) and _goBackTitle is not same as('') %}
-                    <li class="d-tablet-none">
+                    <li class="{{ _displayTabletNoneClassName }}">
                         <Icon name="chevron-left" />
                         <Link href="{{ item.url }}" color="primary" isUnderlined>{{ _goBackTitle }}</Link>
                     </li>
                 {% endif %}
-                <li class="d-none d-tablet-flex">
+                <li class="{{ [_displayNoneClassName, _displayTabletFlexClassName] | join(' ') }}">
```

Comment:
I think our `classProp()` function can be applied here, but it doesn't matter if the component overhaul is already in progress.

Reply:
Yep, but will be refactored soon.